app/controllers/course/survey/responses_controller.rb

```diff
@@ -77,7 +77,7 @@ def handle_create_error(error)
     @response = @survey.responses.accessible_by(current_ability).
                 find_by(course_user_id: current_course_user.id)
     if @response
-      render json: { responseId: @response.id }, status: :bad_request
+      render partial: 'created', status: :bad_request
```

Comment:
I noticed that you used `found`( 302 ),  any reasons ?  Where in this case I think 303 is more suitable ( for redirecting POST ): https://serverfault.com/questions/391181/examples-of-302-vs-303